swagger/public_v1/template.yml

```
@@ -374,3 +374,31 @@ components:
           example: subjects
         attributes:
           $ref: "#/components/schemas/SubjectAttributes"
+    SubjectAreaResource:
+      description: "This schema provides metadata about a subject area."
+      type: object
+      properties:
+        id:
+          type: string
+          example: "PrimarySubject"
+        type:
+          type: string
+          example: "subject_areas"
+        attributes:
+          $ref: "#/components/schemas/SubjectAreaAttributes"
+        relationships:
+          $ref: "#/components/schemas/SubjectAreaRelationships"
+    SubjectAreaListResponse:
+      description: "This schema is used to return a collection of subject areas."
+      type: object
+      required:
+        - data
```

```suggestion
        - data
        - jsonapi
```

Probably that as well...